### PR TITLE
[cmake][linux] Insert PulseAudio block into kodi-standalone

### PR DESCRIPTION
**Pull request by hudokkow**
Don't know if it's the best place for this but placing it inside FindPulseAudio.cmake didn't felt right.
Opinions welcome.

### CODE REVIEW

File: project/cmake/scripts/linux/install.cmake [hunk 1]
```diff
@@ -19,6 +19,12 @@ configure_file(${CORE_SOURCE_DIR}/project/cmake/${APP_NAME_LC}-config.cmake.in
 
 configure_file(${CORE_SOURCE_DIR}/tools/Linux/kodi.sh.in
                ${CORE_BUILD_DIR}/scripts/${APP_NAME_LC} @ONLY)
+
+if (ENABLE_PULSEAUDIO AND PULSEAUDIO_FOUND)
+  file(READ "${CORE_SOURCE_DIR}/tools/Linux/kodi-standalone.sh.pulse" pulse_content)
+  set (XBMC_STANDALONE_SH_PULSE ${pulse_content})
+endif()
+
```
Comment (fetzerch) [inline project/cmake/scripts/linux/install.cmake]: Code looks good.
(usually we don't have spaces after if or set in cmake, but I won't insist)

**Comment by hudokkow** [comment]
Yeah... the habit and the monk thing.
Corrected and added a check for file existence.

**Comment by hudokkow** [comment]
@MartijnKaijser, not that it matters for this AFAIK but is jenkins bridge down?

**Comment by hudokkow** [comment]
Travis is happy, merging.

**Pull request merged**